config/webpack/partial/pwa.js

```diff
@@ -82,7 +82,11 @@ module.exports = function () {
             favicons: true
           }
         }),
-        new SWPrecacheWebpackPlugin(precacheConfig)
+        new AddManifestFieldsPlugin({
```

Comment:
We should use `manifestConfig` here. We can set defaults there and send it along to this plugin.

https://github.com/Aweary/electrode-archetype-react-app/blob/e01873ca469e2b1ef34a096182d1b9be199373a3/config/webpack/partial/pwa.js#L34